### ISSUE TITLE
"libm.so.6" not found.

### ISSUE THREAD

**Issue by JankleDev** [cabaletta/baritone#4274]
## Some information
Operating system: linux-android
Java version: 17
Minecraft version: 1.20.2
Baritone version: 1.2.19-599-ga223585d
Other mods (if used): None

## How to reproduce
Add baritone-unoptimized-fabric-1.2.19-599-ga223585d;
Load up game in an linux-android.

## Modified settings
To get the modified settings run `#modified` in game

## Final checklist
- [x] I know how to properly use check boxes 
- [✓] I have included the version of Minecraft I'm running, baritone's version and forge mods (if used).
- [x] I have included logs, exceptions and / or steps to reproduce the issue.
- [✓] I have not used any OwO's or UwU's in this issue.

**Comment by JankleDev** [comment]
```
[21:20:25] [Render thread/INFO]: [STDOUT]: [nether-pathfinder] Created temp file at /data/user/0/net.kdt.pojavlaunch.debug/cache/libnether_pathfinder-aarch644909904464162195681.so
[21:20:25] [Render thread/INFO]: [STDERR]: [nether-pathfinder] Failed to load shared library
[21:20:25] [Render thread/INFO]: [STDERR]: java.lang.UnsatisfiedLinkError: /data/data/net.kdt.pojavlaunch.debug/cache/libnether_pathfinder-aarch644909904464162195681.so: dlopen failed: library "libm.so.6" not found: needed by /data/data/net.kdt.pojavlaunch.debug/cache/libnether_pathfinder-aarch644909904464162195681.so in namespace clns-4
```

**Comment by ZacSharp** [comment]
This means the JVM can't find a native library needed by the elytra pathfinder (probably doesn't exist because you're using a phone). Maybe someone on the PojavLauner repository / discord can help with that.

**Comment by JankleDev** [comment]
I think it's because `libc.so.6` and `libm.so.6` are made for Gnu/Linux not Android/Linux. I think it's not possible to cross-compile it either.

**Comment by JankleDev** [comment]
Elytra Pathfinder was added in 1.20 update, it would be great if they could just separate them.

**Comment by ZacSharp** [comment]
Baritone should work fine anyways, just without elytra capabilities.

**Comment by ZacSharp** [comment]
Also elytra support is not exclusive to Baritone for 1.20.*, it is included in the last release for 1.12 (1.2.19) and is present in the code for 1.19.4 and 1.20.4.

**Comment by JankleDev** [comment]
babbaj's nether-pathfinder is causing the issue. I've tested 1.19.2 and it works fine.

**Comment by ZacSharp** [comment]
As I said, Baritone should work fine regardless of the error message. If the game is crashing or Baritone is not working please say so explicitly since right now to me it looks like nether-pathfinder gracefully detected that it can't work on your system and in that case Baritone will simply not try to use it.

**Comment by ZacSharp** [comment]
The only part which is not supposed to work is `#elytra`, which will tell you that your system is not supported instead of it's normal actions.

**Comment by wagyourtail** [comment]
Actually its broken in the release and pojav crashes. This fixed it https://github.com/cabaletta/baritone/pull/4244

**Comment by ZacSharp** [comment]
Ehm right. Forgot about that one still being a thing.

**Comment by JankleDev** [comment]
> Actually its broken in the release and pojav crashes. This fixed it #4244

All 1.20 releases are broken? Do I need to build it myself?

**Comment by ZacSharp** [comment]
Yes, and since the changes are not merged up yet (~if things go to plan I'll do that in a couple days~ see edit below) you'll also have to apply the change from #4244 to your source set before building. It's a single word change with no version dependencies so no big deal.
EDIT: You can also build from #4280 or use its CI build.
EDIT2: Since that pr is merged now, any build from the 1.20.4 branch will do.
